cpp/arcticdb/util/test/test_storage_lock.cpp

```diff
@@ -182,6 +182,46 @@ struct ForceReleaseLockTask {
     }
 };
 
+struct OptimisticForceReleaseLockTask {
+    std::shared_ptr<LockData> data_;
+    size_t timeout_ms_;
+    size_t retry_ms_;
+
+    OptimisticForceReleaseLockTask(std::shared_ptr<LockData> data, size_t timeout_ms, size_t retry_ms) :
+            data_(std::move(data)),
+            timeout_ms_(timeout_ms),
+            retry_ms_(retry_ms)
+    {
+    }
+
+    folly::Future<folly::Unit> operator()() {
+        std::this_thread::sleep_for(std::chrono::milliseconds(1));
```

Comment:
Why is there a sleep here?

Reply:
Again I'm not sure, just how these unit tests were originally implemented. Agree we should remove then?

Reply:
Seems unnecessary to me. Maybe someone with more context can take a look as well.

Reply:
I think fine to remove. The tests still pass and there's no reason to have them.